Sources/Machine/ffnn.hpp

```diff
@@ -135,13 +135,18 @@ class FFNN : public AbstractMachine {
     }
   }
 
-  void InitRandomPars(int seed, double sigma) override {
+  void InitRandomPars(double sigma, nonstd::optional<unsigned> seed) override {
+    if (!seed.has_value()) {
+      std::random_device rd;
```

Comment:
very dangerous, different processors might end up with different initial parameters for the wave function, to be removed/improved...

Reply:
```
seed=1234
```
is a better option actually